src/lib/chat/sendbird-client.ts

```diff
@@ -71,6 +71,16 @@ export class SendbirdClient implements IChatClient {
     }
   }
 
+  async getConversations() {
+    try {
+      const conversations = await get<any>('/directMessages/mine');
+      return (conversations.body || []).map((currentChannel) => toLocalChannel(currentChannel));
```

Comment:
```suggestion
      return (conversations.body || []).map(toLocalChannel);
```
You can pass the mapping function directly to `map`. There's no need to wrap it with a lambda.